worker/buildbot_worker/pbutil.py

```diff
@@ -68,15 +63,18 @@ def clientConnectionLost(self, connector, reason):
         RCF = protocol.ReconnectingClientFactory
         RCF.clientConnectionLost(self, connector, reason)
 
-    def startedConnecting(self, connector):
-        self.startHungConnectionTimer(connector)
-
     def clientConnectionMade(self, broker):
         self.resetDelay()
         PBClientFactory.clientConnectionMade(self, broker)
         self.doLogin(self._root, broker)
         self.gotRootObject(self._root)
 
+    def buildProtocol(self, addr):
+        return PBClientFactory.buildProtocol(self, addr)
+
+    def _hung_connection(self):
```

Comment:
Is this method used? I see a reference to `_hung_connection` in `Worker` but it has a `_hung_connection` method.